lib/rucio/tests/test_meta.py

```diff
@@ -8,18 +8,23 @@
 # Authors:
 # - Vincent Garonne, <vincent.garonne@cern.ch>, 2012-2013
 
-from nose.tools import assert_true, assert_in, raises, assert_is_instance
+from nose.tools import assert_equal, assert_in, assert_is_instance, assert_true, raises
 
-from rucio.db.sqla.constants import DIDType
+from rucio.client.didclient import DIDClient
 from rucio.client.metaclient import MetaClient
-from rucio.common.exception import KeyNotFound, InvalidValueForKey, UnsupportedValueType
+from rucio.common.exception import InvalidValueForKey, KeyNotFound, UnsupportedValueType
 from rucio.common.utils import generate_uuid as uuid
+from rucio.db.sqla.constants import DIDType
 
 
 class TestMetaClient():
 
     def setup(self):
         self.meta_client = MetaClient()
+        self.did_client = DIDClient()
+        self.tmp_scope = 'mock'
+        self.tmp_name = 'name_%s' % uuid()
+        self.did_client.add_did(scope=self.tmp_scope, name=self.tmp_name, type="DATASET", statuses={'monotonic': True})
```

Comment:
no need for monotonic.

(monotonic just means that you're not allowed to detach files from the dataset)